maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/ModuleInstaller.java

```diff
@@ -50,27 +50,28 @@ public ModuleInstaller(MavenProject mavenProject,
         this.versionsHelper = versionsHelper;
     }
 
-    public void installCoreModules(Server server, boolean isCreatePlatform, DistroProperties properties, DistroHelper distroHelper) throws MojoExecutionException, MojoFailureException {
+    public void installDefaultModules(Server server) throws MojoExecutionException {
+        boolean isPlatform = server.getVersion() == null;  // this might be always true, in which case `getCoreModules` can be simplified
+        List<Artifact> coreModules = SDKConstants.getCoreModules(server.getPlatformVersion(), isPlatform);
+        if (coreModules == null) {
+            throw new MojoExecutionException(String.format("Invalid version: '%s'", server.getPlatformVersion()));
+        }
+        installModules(coreModules, server.getServerDirectory().getPath());
+    }
+
+    public void installModulesForDistro(Server server, DistroProperties properties, DistroHelper distroHelper) throws MojoExecutionException {
```

Comment:
...and this function is the corresponding "if" section of the old `installCoreModules` function.